app/controllers/api/articles_controller.rb

```diff
@@ -42,10 +42,14 @@ def update
 
   private
 
+  def attach_image(article)
+    params[:article][:image].present? && DecodeService.attach_image(params[:article][:image].first, article.image)
```

Comment:
We are sending the image as a string (not an array), so the `first` will fack it up :P

Reply:
ok, fixed this